test/BeePayments.test.js

```diff
@@ -271,3 +271,78 @@ contract('BeePayments Arbitration Test', function (accounts) {
     });
 
 });
+
+contract('BeePayments Arbitration Guest Test', function (accounts) {
+    // account[0] points to the owner on the testRPC setup
+    var owner = accounts[0];
+    var user1 = accounts[1];
+    var demand = accounts[2];
+    var supply = accounts[3];
+    var uuid = "x";
+
+
+    beforeEach(function () {
+        return BeePayments.deployed().then(function (instance) {
```

Comment:
Let's start deploying contracts within the tests and not using contracts deployed outside.